src/model.ts

```diff
@@ -1,78 +1,85 @@
 export type WithTimestamp = {
-    timestamp: string
-}
+  timestamp: string;
+};
 
-export type WithKeyword<T> = { [P in keyof T & string as `${P}.keyword`]: T[P] };
+export type WithKeyword<T> = {
+  [P in keyof T & string as `${P}.keyword`]: T[P];
+};
 
 export enum SortOrder {
-    Desc = 'desc',
-    Asc = 'asc'
+  Desc = 'desc',
+  Asc = 'asc',
 }
 
 export type Hash = {
-    hash: string
-}
+  hash: string;
+};
 
 export type Ordinal = {
-    ordinal: number
-}
+  ordinal: number;
+};
 
 export type BalanceValue = {
-    balance: number
-}
+  balance: number;
+};
 
-export type Balance = Ordinal & BalanceValue
+export type Balance = Ordinal & BalanceValue;
 
 type Height = {
-    min: number,
-    max: number
-}
+  min: number;
+  max: number;
+};
 
 export type BaseSnapshot = {
-    height: number,
-    subHeight: number,
-    lastSnapshotHash: string
-} & WithTimestamp & Ordinal & Hash
+  height: number;
+  subHeight: number;
+  lastSnapshotHash: string;
+} & WithTimestamp &
+  Ordinal &
+  Hash;
 
-export type BaseBlock = Hash
+export type BaseBlock = Hash;
 
 export type OpenSearchSnapshot = {
-    blocks: BaseBlock[]
-} & BaseSnapshot
+  blocks: string[];
+} & BaseSnapshot;
 
 export type Snapshot = {
-    blocks: string[]
-} & BaseSnapshot
+  blocks: string[];
+} & BaseSnapshot;
 
 export type RewardTransaction = {
-    destination: string,
-    amount: number
-}
+  destination: string;
+  amount: number;
+};
 
 export type WithRewards = {
-    rewards: RewardTransaction[]
-}
+  rewards: RewardTransaction[];
+};
 
 export type OpenSearchTransaction = {
-    hash: string
-}
+  hash: string;
+};
 
 export type OpenSearchBlock = {
-    hash: string,
-    height: number,
-    transactions: OpenSearchTransaction[],
-    parent: BlockReference[]
-}
+  hash: string;
+  height: number;
+  transactions: string[];
+  parent: BlockReference[];
+  snapshotHash: string;
+  snapshotOrdinal: number;
+} & WithTimestamp;
 
 export type BlockReference = {
-    hash: string,
-    height: number
-} 
+  hash: string;
+  height: number;
+};
 
 export type Block = {
-    hash: string,
-    height: number,
-    transactions: string[],
-    parent: BlockReference[],
-    snapshot: string,
-} & WithTimestamp
+  hash: string;
+  height: number;
+  transactions: string[];
+  parent: BlockReference[];
+  snapshotHash: string;
+  snapshotOrdinal: number;
```

Comment:
There is no such field in API DOC proposal. Should we change it here or in API DOC?